pages/goals/[slug].tsx

```diff
@@ -76,11 +76,26 @@ const EditGoalsPage = () => {
                 }));
               }}
             />
+
             {editedGoalValues.goalName.length > 60 && (
               <p className="text-xs text-red-600">
                 please keep your goal under 60 characters
               </p>
             )}
+            <p className="font-bold">Write An Actionable Goal Outline</p>
```

Comment:
purely editorial suggestion here - what if we stick to "goal notes"? it seems less prescriptive i.e write whatever you want here associated with the goal.

what if we put "e.g write an actionable goal outline" in the placeholder if notes are empty instead?

Reply:
that's even better. just made the change